CMakeLists.txt

```diff
@@ -80,7 +85,7 @@ set(third_party "${CMAKE_SOURCE_DIR}/3rdparty/")
 set(EXTERNAL_DEPS "")
 
 if(BUILD_TESTING)
-  add_subdirectory("${third_party}/googletest")
+  #add_subdirectory("${third_party}/googletest")
```

Comment:
I suppose you commented this out because you didn't find googletest when you compiled...

That's because googletest is a submodule in this configuration. You'll have to explicitly donwload it with git.

From inside the directory run:

`$ git submodule update --init`

Or clone the repo again, but this time like this:

`$ git clone <...> --recursive`

Reply:
Done. Thanks